com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/CreateOrUpdateAppStep.java

```diff
@@ -245,6 +251,14 @@ public void handleApplicationAttributes() {
 
         @Override
         public void handleApplicationServices() throws FileStorageException {
+            Boolean passedTestingPhase = (Boolean) execution.getContext()
+                                                            .getVariable(Constants.VAR_PASSED_TESTING_PHASE);
+            if (processTypeParser.getProcessType(execution.getContext())
+                                 .equals(ProcessType.BLUE_GREEN_DEPLOY) &&
+                BooleanUtils.toBooleanDefaultIfNull(passedTestingPhase, false)) {

```

Comment:
Why not handle this in a getter in `StepsUtil`. There we already have a generic method that allows you to provide a default value.

Reply:
Changed to `StepsUtil.getBoolean(context, varName, defaultValue)` and removed check for blue-green deploy process id